radio/src/targets/horus/board.cpp

```diff
@@ -89,11 +87,19 @@ void interrupt1ms()
   DEBUG_TIMER_START(debugTimerRotEnc);
   checkRotaryEncoder();
   DEBUG_TIMER_STOP(debugTimerRotEnc);
+
+#if defined(LUA)
+  if (telemetryProtocol == PROTOCOL_FRSKY_SPORT) {
+    if (telemetryFifo.last(-2) == 0x7E && telemetryFifo.last(-1) == outputTelemetryBufferTrigger) {
+      sportSendBuffer(outputTelemetryBuffer, outputTelemetryBufferSize);
```

Comment:
Confirmed, not related to the S6R nor even a receiver in any way, the telemetry just hangs after a few minutes. If I power the radio on with no receiver powered, wait 5 minutes, then power up the receiver I have no telemetry.